airflow/providers/openlineage/sqlparser.py

```diff
@@ -299,7 +304,8 @@ def create_information_schema_query(
         is_cross_db: bool,
         information_schema_columns,
         information_schema_table,
```

Comment:
Any chance you could add these two type hints as well while you are in there?

Reply:
Added. Not sure why I skipped these.